sds_data_manager/stacks/monitoring_stack.py

```diff
@@ -0,0 +1,26 @@
+from aws_cdk import Stack, aws_sns
+from constructs import Construct
+
+
+class MonitoringStack(Stack):
+    """Monitoring stack
+
+    Set up an integration with Slack to send notifications to a channel.
+
+    This had to be done manually in the AWS console to authorize the
+    AWS account to use the Slack app.
+    Just follow the steps here to create a new Slack authorization.
+    https://us-east-2.console.aws.amazon.com/chatbot/home
+    Choosing the channel and permissions as you go through the steps.
```

Comment:
Thanks for adding these instructions here!

Reply:
Moved this info up to the module level now.